### PR TITLE
[META-59] Integrate backend data (logo, owner, eli5)

### PR DESCRIPTION
**Pull request by kigadye**
### Description of issue
Projects table and detail page now show actual logos, owners and eli5 description

### How I implemented
- Updated queries & ran codegen to regenerate types/ hooks
- Adapted table columns and detail page to use actual data
- Fixed outdated codegen config

### Other
Screenshot 2023-06-11 at 07 58 19

Screenshot 2023-06-11 at 07 58 42

### CODE REVIEW

File: src/components/page/details/ProjectInformation.tsx [hunk 1]
```diff
@@ -17,15 +18,16 @@ const ProjectInformation = ({ image, url, name, eli5, tags }: ProjectInformation
...
-        {/* @TODO Replace with actual image */}
-        <div className="mr-4 h-[30px] w-[30px] rounded-[5px] bg-gray-600" />
+        <div className="relative mr-4 h-8 w-8 overflow-hidden rounded-[5px] bg-gray-600">
+          <Image src={image} alt="logo" fill sizes="32px" />
+        </div>
 
-        <h1 className="mr-3 text-20 font-medium">
-          <a href={url} target="_blank" rel="noreferrer">
+        <a href={url} target="_blank" rel="noreferrer">
+          <h1 className="mr-3 text-20 font-medium">
```
Comment (ArnoClaude) [inline src/components/page/details/ProjectInformation.tsx:26]: At first, I clicked the name expecting to get to the user's GH profile. Can we separate the user name and repo both visually and when clicking on them?

```
const getGitHubUserUrl = (url: string) => {
  const urlSplit = url.split('/')
  return `${urlSplit[0]  }//${  urlSplit[2]  }/${  urlSplit[3]}`
}

const getGitHubUserName = (name: string) => {
  const nameplate = name.split('/')
  return nameSplit[0]
}

const getGitHubRepoName = (name: string) => {
  const nameSplit = name.split('/')
  return nameSplit[1]
}
``` 

...

```
<a href={getGitHubUserUrl(url)} target="_blank" rel="noreferrer">
          <h1 className="mr-3 text-20 font-light text-gray-500">
            {getGitHubUserName(name).slice(0, 32)}
            {getGitHubUserName(name).length > 32 ? '...' : ''}
          </h1>
</a>

<a href={url} target="_blank" rel="noreferrer">
          <h1 className="mr-3 text-20 font-medium">
            {getGitHubRepoName(name).slice(0, 32)}
            {getGitHubRepoName(name).length > 32 ? '...' : ''}
          </h1>
</a>
```

File: src/components/page/details/ProjectInformation.tsx [hunk 2]
```diff
@@ -17,15 +18,16 @@ const ProjectInformation = ({ image, url, name, eli5, tags }: ProjectInformation
...
             {name.slice(0, 32)}
             {name.length > 32 ? '...' : ''}
```
Comment (ArnoClaude) [inline src/components/page/details/ProjectInformation.tsx:28]: If you want, you can also define a max width for this div and then let the words automatically break with a `...` at the end using [Tailwind's text-ellipsis overflow](https://tailwindcss.com/docs/text-overflow).

**Review by ArnoClaude — APPROVED** [review]
Great job. Left 2 comments, which might improve the UX/code a bit :)

**Review by DamianGrosz2 — APPROVED** [review]
